examples/request_state_local_cache/src/tests.rs

```diff
@@ -0,0 +1,15 @@
+use std::sync::atomic::{Ordering};
+
+use ::Atomics;
+use super::rocket;
+use rocket::local::Client;
+
+#[test]
+fn test() {
+    let client = Client::new(rocket()).unwrap();
+    client.get("/").dispatch();
+
+    let atomics = &client.rocket().state::<Atomics>().unwrap();
```

Comment:
I don't think you need the `&` there.

Reply:
Hmh, when I tried without `&`, I had to de-reference (`(*atomics).cached...`) the value when I wanted to access it for some reason. Seems to work without one now.